vale/styles/jargon/Suggestions.yml

```diff
@@ -0,0 +1,20 @@
+---
+extends: substitution
```

Comment:
A few questions:
- What is the schema for the files?
- Do you need to use Regular Expressions? Would simple conventions work instead?
  - Word breaks are assumed: 
    - start of word: `/(?<=^|\P{L})(?=\p{L})/gmu` (these are not exactly correct)
    - end of word: `/(?<=\p{L})(?=\P{L}|$)/gmu`
  - A space between words will match one or more spaces.
- How do you hope to use these files?

Reply:
These are rules for vale tool

https://vale.sh

Reply:
I'll check it out.